polus-tiledtiff-converter-plugin/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM openjdk:8-alpine
```

Comment:
You should use a [multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) to build your maven package. That way the maven project will also be recompiled in the CI pipeline on new code changes.